### ISSUE TITLE
List of apps resets somehow

### ISSUE THREAD

**Issue by buergi** [plusCubed/velociraptor#120]
I haven't figured out when exactly, but on an Galaxy S7 with Android 8 I'm having the problem that somehow the list of apps in which Velociraptor is shown often gets reset, so all map apps are selected again.
I think it might have something to do with automatic updating of apps from the play store. I suspect that the updating of an app somehow causes this issue. However, I can't always reproduce it.
However, every few days the list of apps is all selected again and velociraptor permanently keeps popping up in any map app, which is pretty annoying.
Is anyone experiencing similar issues?

**Comment by gordyhike** [comment]
Yes, I am experiencing the same issues. It pops up on applications I haven't chosen. I turn them off, but it returns a few days later. FYI.. my phone is a Oneplus 6T.

**Comment by Qwerty-uk** [comment]
Also getting the same thing, on LG G6, so it's clearly a general bug and not device-dependent.

**Comment by CoolHandLouis** [comment]
Same on Moto G7 Power

**Comment by mattsches** [comment]
Same here, using latest 1.7.0 version. Seems to happen after rebooting the phone (which I've been doing quite often recently).

**Comment by hmueller01** [comment]
I can reproduce this after rebooting the device. Android 12. If you force terminating the app it happens as well. Seems these data are not correctly saved or resorted ...

**Comment by Juhaniakis** [comment]
Same here. Android 12 with Samsung S20 FE phone and Android 10 with Atoto S8 Premium car player  . I want to use this with Car Launcher Pro and after every boot app selections reset.

**Comment by M-Buerger** [comment]
Same problem for me too. Using Samsung A71 / Android 12. I can't reproduce the bug,  seems to happen randomly. Very unhappy that this is still unsolved, love the app, but this bug is annoying.

**Comment by nnn406** [comment]
The same for me

**Comment by theLSB** [comment]
Yeap, same for me, exactly as the OP described it.
Pixel 4a (4G) Android 13
It really annoying, since some apps that are not navigation apps but that use GPS are listed as well, like Threema.

**Comment by ErdbeerbaerLP** [comment]
I made a fork fixing this particular issue
https://github.com/ErdbeerbaerLP/velociraptor/releases

Tested on my Mi 10T pro and Galaxy A70 (both with AOSP ROMs)

Edit: fixed wrong apk upload

**Comment by gordyhike** [comment]
@ErdbeerbaerLP 
The fork works for me, Thanks
My phone is a Oneplus 6T.
The first apk wouldn't install, but the update works great, Thank you

**Comment by Kitt3120** [comment]
> I made a fork fixing this particular issue
> https://github.com/ErdbeerbaerLP/velociraptor/releases
> 
> Tested on my Mi 10T pro and Galaxy A70 (both with AOSP ROMs)
> 
> Edit: fixed wrong apk upload

True hero
